scripts/release/release.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/rogpeppe/go-internal/semver"
+)
+
+func main() {
+	r := parseArgs()
+
+	r.validateSemver()
+	r.cloneRepository()
+	r.createGitTag()
+	r.createGitHubRelease()
+	r.publishImageInQuay()
+}
+
+func parseArgs() *release {
+	flag.Set("logtostderr", "true")
+	flag.Set("stderrthreshold", "WARNING")
+	flag.Set("v", "2")
+
+	version := flag.String("version", "", "Release version tag. Must be a valid semver. The branch is automatically detected from the major and minor release")
+	githubTokenFile := flag.String("github-token-file", "", "File containing the github token.")
+	quayUsername := flag.String("quay-username", "", "Quay username.")
+	quayTokenFile := flag.String("quay-token-file", "", "File containing the quay token.")
+	flag.Parse()
+
+	if *githubTokenFile == "" {
```

Comment:
What is the parameter is missing? nil or pointer to an empty string?

Reply:
it's an empty string because of the default value given by the `flag.String` function